File: urbano-norte-mvp/mobile/app/src/main/java/com/urban/mobile/model/Anuncio.kt
package com.urban.mobile.model
data class Anuncio(
    val id: String,
    val title: String,
    val description: String,
    val images: List<String>,
    val available: Boolean,
    val user: User2,
    val pacote: List<Pacote>
)
data class User2(
    val id: String,
    val name: String,
    val phone: String,
)
data class Pacote(
    val id: String,
    val name: String,
    val description: String,
    val price: Double,
    val available: Boolean,
    val addTime: Int
)

File: urbano-norte-mvp/backend/models/Anuncio.js
const mongoose = require('../db/conn');
const { Schema } = mongoose;

const AnuncioSchema = new Schema({
  title: {
    type: String,
    required: true,
  },
  description: {
    type: String,
  },
  images: {
    type: Array,
    required: true,
  },
  available: {
    type: Boolean,
  },
  user: {
    type: Schema.Types.Mixed, 
    required: true
  },
  company: {
    type: Schema.Types.Mixed, 
    required: true
  },
  pacote: {
    type: Array, 
    required: true
  },
}, { timestamps: true });

const Anuncio = mongoose.model('Anuncio', AnuncioSchema);

module.exports = Anuncio;
